core/src/main/java/hudson/model/AbstractBuild.java

```diff
@@ -733,8 +734,10 @@ protected final boolean performAllBuildSteps(BuildListener listener, Iterable<?
 
         private void reportError(BuildStep bs, Throwable e, BuildListener listener, boolean phase) {
             String msg = "Publisher " + bs.getClass().getName() + " aborted due to exception";
-            e.printStackTrace(listener.error(msg));
-            LOGGER.log(WARNING, msg, e);
+            if (!(e instanceof AbortException)){
```

Comment:
If it _is_ an `AbortException`, is the message being printed somewhere else already? Anyway

> aborted due to exception

does not sound right in this context.